<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="LPR">
    <attribute name="title">
     <string>LPR</string>
    </attribute>
    <widget class="QPushButton" name="OpenGateButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CloseGateButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>close</string>
     </property>
    </widget>
    <widget class="QLabel" name="LPVideo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="text">
      <string>Video</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicensePlateInformation_1">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicensePlateInformation_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicensePlateInformation_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicensePlateInformation_4">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicensePlateInformation_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>50</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicensePlateInformation_6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicensePlateInformation_0">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicensePlateInformation_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="AutoCar">
    <attribute name="title">
     <string>AutoCar</string>
    </attribute>
    <widget class="QPushButton" name="LoadVideoButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StopVideoButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QLabel" name="Video">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
     <property name="text">
      <string>Video</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ForwardButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>164</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RightButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LeftButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>L</string>
     </property>
    </widget>
    <widget class="QLabel" name="SteerWheel">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>206</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>方向盘</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
